BLSQ/iaso
# IA-2753:Pictures of health facilities should look like real ones

Related JIRA tickets : [IA-2753](https://bluesquare.atlassian.net/browse/IA-2753)

## Self proofreading checklist



## Changes

Replaced fake health facility images with real one when creating submissions!

## How to test

From setuper folder, launch `python3 setuper.py` to create data and open submissions page on the file tab : [http://localhost:8081/dashboard/forms/submissions/tab/files](http://localhost:8081/dashboard/forms/submissions/tab/files)

You should see real health facility images!

## Print screen / video


https://github.com/BLSQ/iaso/assets/6383219/b20ed330-b9c8-4e9f-a967-cb55498cd748




[IA-2753]: https://bluesquare.atlassian.net/browse/IA-2753?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
